theano/scalar/basic.py

```diff
@@ -852,6 +852,7 @@ class ScalarOp(Op):
 
     nin = -1
     nout = 1
+    __props__ = ("output_types_preference", "name")
```

Comment:
@abergeron what do you think of adding props to Scalar? I think it is fine, but as we didn't do it before, a look by someone else would be good.